pypowerwall/local/pypowerwall_local.py

```diff
@@ -5,6 +5,8 @@
 
 import requests
 
+import pypowerwall.local.tesla_pb2 as tesla_pb2
```

Comment:
I moved this protobuf librayr into local since it only applies to local access for systems on Firmware < 23.44.0.  
I'm still on an older Firmware so I can still test.

Reply:
Works for me